client/web/src/search/results/StreamingSearchResults.tsx

```diff
@@ -185,12 +191,18 @@ export const StreamingSearchResults: FC<StreamingSearchResultsProps> = props =>
             if (results.results.length > 0) {
                 telemetryService.log('SearchResultsNonEmpty')
             }
+
+            // setTimeout avoids a "Cannot update a component while rendering a different component" React error
```

Comment:
nit: slight rewording of the comment
```suggestion
            // Add the recent search in the next queue execution to avoid
            // updating a React component while rendering another component.
```